app/assets/javascripts/notebook/magic/tsChart.coffee

```diff
@@ -12,6 +12,8 @@ define([
     chart_container.attr("id", "custom-c3-chart-"+@genId).appendTo(container)
 
     data = {
+#      https://github.com/d3/d3-time-format/blob/master/README.md#locale_format
+      xFormat: '%a %b %d %H:%M:%S %Y %Z'
     }
```

Comment:
P.S. note to self: this refers to [`TimeseriesChart(dataRows)`](https://github.com/spark-notebook/spark-notebook/blob/a746b48b7ac376f2e8edea73e50d435640620308/modules/common/src/main/scala/notebook/front/widgets/charts/Defaults.scala#L122-L134)